typescript/src/lib/bitcoin/hash.ts

```diff
@@ -4,21 +4,19 @@ import { Hex } from "../utils"
 /**
  * Computes the HASH160 (i.e. RIPEMD160(SHA256(text))) for the given text.
  * @param text Text the HASH160 is computed for.
- * @returns Hash as a 20-byte un-prefixed hex string.
+ * @returns 20-byte-long hash.
  */
-function computeHash160(text: string): string {
-  const sha256Hash = utils.sha256(
-    Hex.from(Buffer.from(text, "hex")).toPrefixedString()
-  )
+function computeHash160(text: Hex): Hex {
+  const sha256Hash = utils.sha256(text.toPrefixedString())
   const hash160 = utils.ripemd160(sha256Hash)
 
-  return Hex.from(hash160).toString()
+  return Hex.from(hash160)
 }
 
 /**
  * Computes the double SHA256 for the given text.
- * @param text Text the double SHA256 is computed for.
- * @returns Hash as a 32-byte un-prefixed hex string.
+ * @param text - Text the double SHA256 is computed for.
```

Comment:
Nitpick, but I think we can get rid of the dash between parameter name and description. No need to change that for the existing code but let's not introduce that in the new code.

Reply:
Done in d698946e8af08f8496c982902c5a47ddadfec330.